hawkbit-ddi-api/src/main/java/org/eclipse/hawkbit/ddi/json/model/DdiStatus.java

```diff
@@ -31,6 +32,7 @@
     @Valid
     private final DdiResult result;
 
+    @Size(min = 0, max = 50)
```

Comment:
Ok, understood. However, if you do it that way its basically not configurable. But I think we can keep for now. 

If its ok for you I would later move this into the QuotaManagement service so that hawkBit operators can customise this with a property.